management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java

```diff
@@ -0,0 +1,16 @@
+package nearsoft.skytouch.management.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.Map;
+
+@Controller
+public class HomeController {
```

Comment:
Is it really needed for this controller to be public?.
Comment: Be careful about the classes', and their attributes, privacy scope.

Reply:
Please read [java specification for class accessor modifiers.](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html)

_At the top level—public, or package-private (no explicit modifier)._